package.json

```diff
@@ -11,9 +11,12 @@
     "format:fix": "prettier --write --list-different ."
   },
   "dependencies": {
+    "@next/font": "^14.1.4",
+    "classnames": "^2.5.1",
     "next": "14.1.4",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "typed-css-modules": "^0.9.1"
```

Comment:
Two comments.

1. `typed-css-modules` should be a dev dependencies.
2. You have a mix of dependencies using `^` and using exact versions. Be consistent and use `^` everywhere.

Reply:
Ok
[x] Done